Fix Issue #13671: Make sections consistent across modules (#15054)
mdn/content: files/en-us/learn/css/building_blocks/images_media_form_elements/index.md

## Test your skills!

You've reached the end of this article, but can you remember the most important information? You can find some further tests to verify that you've retained this information before you move on — see [Test your skills: Images and form elements](/en-US/docs/Learn/CSS/Building_blocks/Images_tasks).

## Summary

This lesson has highlighted some of the differences you will encounter when working with images, media, and other unusual elements in CSS.

> In the next article, we'll look over a few tips you'll find useful when you have to [style HTML tables](/en-US/docs/Learn/CSS/Building_blocks/Styling_tables).

Reviewer: ```suggestion
In the next article, we'll learn how to [style HTML tables](/en-US/docs/Learn/CSS/Building_blocks/Styling_tables).
```
